clap_generate/src/generators/mod.rs

```diff
@@ -185,7 +185,7 @@ pub trait Generator {
     fn flags<'help>(p: &App<'help>) -> Vec<Arg<'help>> {
         debug!("flags: name={}", p.get_name());
 
-        let mut flags: Vec<_> = p.get_flags_no_heading().cloned().collect();
+        let mut flags: Vec<_> = p.get_flags_with_no_heading().cloned().collect();
```

Comment:
Seriously?

Reply:
https://github.com/clap-rs/clap/pull/2114#discussion_r479368160

I thought we had a compromise. But let's continue the discussion.

Reply:
No need. Let's do what: we rename this method, but you owe me one rename 😄 . Deal?

Reply:
Deal. I am really confused with the previous name.